test/functional/cmdLineTests/classesdbgddrext/playlist.xml

```diff
@@ -90,38 +90,4 @@
 			<subset>SE110</subset>
 		</subsets>
 	</test>
-	<test>
-		<testCaseName>cmdLineTester_classesdbgddrext_aix</testCaseName>
-		<variations>
-			<variation>Mode110</variation>
-			<variation>Mode610</variation>
-		</variations>
-		<command>$(MKTREE) $(REPORTDIR); \
-		cd $(REPORTDIR); \
-		perl $(JVM_TEST_ROOT)$(D)TestConfig$(D)scripts$(D)tools$(D)sysvcleanup.pl aix ; \
```

Comment:
This old version passes a different argument to `sysvcleanup.pl`: are we sure we want merge with the other test?
@smlambert ?

Reply:
Not exactly sure why this doesn't show up outdated, but there's a new target for aix and zos, since the script does the same thing for both platforms.

Reply:
Converging the 2 targets into one and using 'all' param to the .pl script makes sense